tensorflow/core/kernels/cwise_ops.h

```diff
@@ -29,6 +29,28 @@ limitations under the License.
 namespace Eigen {
 namespace internal {
 
+#if GOOGLE_CUDA
```

Comment:
This should really be fixed in Eigen instead, e.g. here: 

https://bitbucket.org/eigen/eigen/src/729d33d11c81fd023834a2d61f0f280ba9da48c8/Eigen/src/Core/MathFunctions.h#lines-438